packages/digit_scanner/lib/pages/qr_scanner.dart

```diff
@@ -75,426 +75,402 @@ class _DigitScannerPageState extends LocalizedState<DigitScannerPage> {
+                            padding: const EdgeInsets.fromLTRB(
+                                kPadding, 0, kPadding, 0),
+                            child: DigitElevatedButton(
+                              child: Text(localizations
+                                  .translate(i18.common.coreCommonSubmit)),
+                              onPressed: () async {
+                                if (widget.isGS1code &&
+                                    result.length < widget.quantity) {
+                                  buildDialog();
+                                } else {
+                                  // final bloc =
+                                  //     context.read<SearchBlocWrapper>();
+                                  // final hfBloc =
+                                  //     context.read<SearchReferralsBloc>();
+
+                                  final bloc = context.read<DigitScannerBloc>();
+                                  bloc.add(DigitScannerEvent.handleScanner(
+                                    barCode: state.barCodes,
+                                    qrCode: state.qrCodes,
+                                  ));
+                                  Navigator.of(
+                                    context,
+                                  ).pop();
+                                }
+                              },
+                            ),
+                          ),
                         ),
-                      ),
-                      Expanded(
-                        child: ListView.builder(
-                          itemCount: widget.isGS1code
-                              ? state.barCodes.length
-                              : state.qrCodes.length,
-                          itemBuilder:
-                              (BuildContext context, int index) {
-                            return ListTile(
-                              shape: const Border(),
-                              title: Container(
-                                margin: const EdgeInsets.only(
-                                  left: kPadding,
-                                  right: kPadding,
-                                ),
-                                height: kPadding * 6,
-                                decoration: BoxDecoration(
-                                  color: DigitTheme.instance
-                                      .colorScheme.background,
-                                  border: Border.all(
-                                    color: DigitTheme
-                                        .instance.colorScheme.outline,
-                                    width: 1,
+
+                        Positioned(
+                          bottom: (kPadding * 7.5),
+                          height: widget.isGS1code
+                              ? state.barCodes.length < 10
+                                  ? (state.barCodes.length * 60) + 80
```

Comment:
change to mediaquery

Reply:
Its based on the barcode length , not the mediaquery size